### ISSUE TITLE
EIKY-Kerry Airport-KERRY-IRELAND

### ISSUE THREAD

**Issue by pilot0674** [InfiniteFlightAirportEditing/Airports#4105]
♻️

**Comment by MajorTomMueller** [comment]
Unfortunately you are already assigned to 3 other issues.
Please finalise them all to OR before starting a new one.
Assignment postponed

**Comment by PatricioAY** [comment]
can i take it

**Comment by MBeringuy** [comment]
![Untitled-1](https://user-images.githubusercontent.com/32115399/66009482-fe09f600-e490-11e9-8b46-d1dd08bc6f2f.jpg)
